src/contrib/persistence/Akka.Persistence.Sql.Common/Journal/BatchingSqlJournal.cs

```diff
@@ -1375,9 +1375,19 @@ protected void AddParameter(TCommand command, string paramName, DbType dbType, o
             param.Value = value;
             param.ParameterName = paramName;
             param.DbType = dbType;
+
+            CustomDbParameterSetup(command, param);
+            
             command.Parameters.Add(param);
         }
 
+        /// <summary>
+        /// Override this to customize <see cref="DbParameter"/> creation used for building database queries
+        /// </summary>
+        /// <param name="command"><see cref="DbCommand"/> used to define a parameter in.</param>
+        /// <param name="param">Parameter to customize</param>
+        protected virtual void CustomDbParameterSetup(TCommand command, DbParameter param) { }
```

Comment:
Would it make more sense to call it something like 'OnBeforeParameterAddToCommand`? (ok maybe that one is a bit much). 

Thinking about how we have other things named around 'hookpoints' (i.e. `AroundPreRestart`) here

Reply:
Yeah, this makes sense... Let me update it to `PreAddParameterToCommand`?

Reply:
Does it seem better now @to11mtm ?